bigquery/main.go

```diff
@@ -56,17 +75,47 @@ func (bq *bqNotifier) SetUp(ctx context.Context, cfg *notifiers.Config, _ notifi
 	if err != nil {
 		return fmt.Errorf("failed to make a CEL predicate: %v", err)
 	}
-	_, ok := cfg.Spec.Notification.Delivery["table"].(string)
+	parsed, ok := cfg.Spec.Notification.Delivery["table"].(string)
 	if !ok {
 		return fmt.Errorf("Expected table string: %v", cfg.Spec.Notification.Delivery)
 	}
 
+	// Initialize client
 	bq.filter = prd
 	bq.client, err = bigquery.NewClient(ctx, projectID)
 	if err != nil {
 		return fmt.Errorf("Failed to initialize bigquery client: %v", err)
 	}
 
+	// Extract dataset id and table id from config
+	rgp, _ := regexp.Compile(".*/.*/.*/(.*)/.*/(.*)")
```

Comment:
Fix: Declare this at the top via `MustCompile` since the contents are static.

Reply:
Resolved!